dotnet/src/extensions/Azure/Handlers/Program.cs

```diff
@@ -1,29 +1,64 @@
-using System.Threading.Tasks;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Azure.Functions.Worker.Configuration;
-using System.Reflection;
 using System.IO;
-using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
 using GeneXus.Deploy.AzureFunctions.Handlers.Helpers;
+using GxClasses.Web;
+using GxClasses.Web.Middleware;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 
 namespace GeneXus.Deploy.AzureFunctions.Handlers
 {
-    public class Program
+	public class Program
```

Comment:
Yes, I used Program because it's the way it's used in the azure doc
https://docs.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#start-up-and-configuration